__tests__/app.test.js

```diff
@@ -37,3 +38,43 @@ describe("/api/topics", () => {
       });
   });
 });
+
+describe("/api/articles", () => {
+  it("200: should respond with an array of article objects with the following properties: author, title, article_id, topic, created_at, votes, comment_count, sorted by created date in decending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Array);
```

Comment:
add another lil expect to this test for the length or the response array to be either greater than zero or equal to what we expect to avoid the chance of any false positives in the case of an empty array!